radicle-cli/examples/rad-patch.md

```diff
@@ -57,6 +57,7 @@ $ rad patch
 
 define power requirements 15141cf1497 R0 3e674d1 (flux-capacitor-power) ahead 1, behind 0
 └─ * opened by z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi (you) [..]
+└─ * patch id d4ef85f57a849bd845915d7a66a2192cd23811f6
```

Comment:
How come the id looks different from the short id above?

Reply:
well, maybe I just copied and paste the wrong one? let me take a look

Reply:
Good catch, I just copied the wrong id!